.gitignore

```diff
@@ -1,9 +1,43 @@
+*.swp
+*.*~
+project.lock.json
```

Comment:
Keep `.gitignore` simple, add only if it necessary. Avoid big universal `.gitignore`.

Reply:
I suggest this minimum:

```
*.user
*.opencover.xml
*.orig
*.nupkg
*.snupkg
.idea
bin
obj
```

Because you use Windows, should be enough just add `.vs`, right?

```
*.user
*.opencover.xml
*.orig
*.nupkg
*.snupkg
.idea
.vs
bin
obj
```

Reply:
@phnx47 
I have just kept following. We can update later when required.

.idea
.vs
bin
obj

Reply:
We still need ignore:
```
*.user
*.opencover.xml
*.orig
*.nupkg
*.snupkg
```

but ok, I can add it later.